ci/build_wheel_cuml.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) 2023-2025, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name="cuml"
+package_dir="python/cuml"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+# TODO(jameslamb): remove this when https://github.com/rapidsai/raft/pull/2531 is merged
+source ./ci/use_wheels_from_prs.sh
+
+# Download the libcuml wheel built in the previous step and make it
+# available for pip to find.
+RAPIDS_PY_WHEEL_NAME="libcuml_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp /tmp/libcuml_dist
+echo "libcuml-${RAPIDS_PY_CUDA_SUFFIX} @ file://$(echo /tmp/libcuml_dist/libcuml_*.whl)" >> /tmp/constraints.txt
+export PIP_CONSTRAINT="/tmp/constraints.txt"
+
+EXCLUDE_ARGS=(
+  --exclude "libcuml++.so"
+  --exclude "libcumlprims_mg.so"
+  --exclude "libcuvs.so"
```

Comment:
Does `libraft.so` need to be on this list? Or `libcumlprims_mg.so`?

Reply:
`libcumlprims_mg.so` already is (1 line up from this comment).

But yes you're right, `libraft.so` probably should be.

```suggestion
  --exclude "libcuvs.so"
  --exclude "libraft.so"
```